src/common/interfaces.ts

```diff
@@ -5,9 +5,18 @@ export interface IConfig {
   has: (setting: string) => boolean;
 }
 
-export interface DumpMetadataOptions {
-  dumpNameFormat: string;
-  stateBucketName?: string;
+export interface LogFn {
+  (obj: unknown, msg?: string, ...args: unknown[]): void;
+  (msg: string, ...args: unknown[]): void;
+}
+
+export interface ILogger {
+  trace?: LogFn;
+  debug: LogFn;
+  info: LogFn;
+  warn: LogFn;
+  error: LogFn;
+  fatal?: LogFn;
 }
```

Comment:
I dont recall in other services we declare the logger log level, doesn't out logger exposes it ?

Reply:
what do you mean by out logger? `jsLogger` doesn't expose any generic interface if that's what you mean